src/fiboat-hash.c

```diff
@@ -0,0 +1,10 @@
+#include <stdint.h>
+
+#include "hash-functions.h"
+
+uint32_t fiboat_hash(const void* data, int len)
+{
+	uint32_t fibo;
+	fibo = fibonacci_32_hash(data, len);
```

Comment:
The idea itself is great

However if you call fibonacci_32_hash() from within the fiboat_hash() function, it will make the performance measurement harder . Same for one-at-a-time.
And the easy way is to forbid inlining to ensure we can get measures for that function.

The functions themselves are small enough for you to copy.  Please do